units/tasks_restart.go

```diff
@@ -0,0 +1,66 @@
+package units
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/util"
+	"github.com/mongodb/amboy"
+	"github.com/mongodb/amboy/job"
+	"github.com/mongodb/amboy/registry"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/logging"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+)
+
+const restartTasksJobName = "restart-tasks"
+
+func init() {
+	registry.AddJobType(restartTasksJobName,
+		func() amboy.Job { return &restartTasksJob{} })
```

Comment:
the job type isn't properly set by default.

Reply:
changed to call the constructor with dummy values